<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SEU IDE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/img/logo.png</normaloff>:/img/img/logo.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件 </string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑 </string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>构建 </string>
    </property>
    <widget class="QMenu" name="menuAssembly">
     <property name="title">
      <string>汇编</string>
     </property>
     <addaction name="actionAssemblyNew"/>
     <addaction name="actionAssemblyAppend"/>
    </widget>
    <addaction name="actionCompile"/>
    <addaction name="menuAssembly"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助 </string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图 </string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="separator"/>
    <addaction name="actionCompileOutPut"/>
    <addaction name="actionAssemblyOutPut"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetCompile">
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Consolas&quot;;</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>编译输出</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="widgetCompileContents">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEditCompileOutput">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetAssembly">
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>汇编输出</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="widgetAssemblyContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEditAssemblyOutput">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/new.png</normaloff>:/img/img/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/open.png</normaloff>:/img/img/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/save.png</normaloff>:/img/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/exit.png</normaloff>:/img/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/undo.png</normaloff>:/img/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/redo.png</normaloff>:/img/img/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/cut.png</normaloff>:/img/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/copy.png</normaloff>:/img/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/paste.png</normaloff>:/img/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/select_all.png</normaloff>:/img/img/select_all.png</iconset>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/find.png</normaloff>:/img/img/find.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/img/replace.png</normaloff>:/img/img/replace.png</iconset>
   </property>
   <property name="text">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionCompileOutPut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>编译输出</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>编译</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionAssemblyOutPut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>汇编输出</string>
   </property>
  </action>
  <action name="actionAssemblyNew">
   <property name="text">
    <string>写入到新文件  </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F9</string>
   </property>
  </action>
  <action name="actionAssemblyAppend">
   <property name="text">
    <string>追加到旧文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F10</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dockWidgetCompile</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionCompileOutPut</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetAssembly</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionAssemblyOutPut</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>976</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToolBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>mainToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
